app/application.py

```diff
@@ -601,7 +609,7 @@ def process_video(video, title, event_date, tags, category, speakers, loc, model
         convert_video_to_mp3(abs_path[:-4] + '.mp4')
         if deepgram or summarize:
             deepgram_data = process_mp3_deepgram(abs_path[:-4] + ".mp3", summarize=summarize, diarize=diarize)
-            result = get_deepgram_transcript(deepgram_data=deepgram_data, diarize=diarize)
+            result = get_deepgram_transcript(deepgram_data=deepgram_data, diarize=diarize, filename=filename)
             if summarize:
                 print("Summarizing")
                 summary = get_deepgram_summary(deepgram_data=deepgram_data)
```

Comment:
on 617 you've changed the function signature but haven't added the third argument.

Reply:
Thanks for the catch, will update in next iteration